src/domains/ecommerce-app/api/ecommerce-io/v1/_get_wallets_by_user.xml.tpl

```diff
@@ -0,0 +1,67 @@
+<policies>
```

Comment:
Is this policy needed? it just call wallet service and return it's response body as is for ok responses. maybe it just be removed and let base policy handle wallet b.e. call

Reply:
ah ok my bad :D